### PR TITLE
requested changes resolved

### PR DESCRIPTION
**Pull request by jstsanam**
Changes done:
- Changed inline styles in tailwind classes
- Adjusted footer to the bottom of the page
- Moved static files to a new directory named 'data'

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ function App() {
   return (
     <>
       <Header />
-      <main className="flex gap-2" style={{ height: "29rem" }}>
+      <main className="flex gap-2 h-[29rem]">
```
Comment (som-matrix) [inline src/App.tsx]: let's use the heights provided by tailwindcss -> https://tailwindcss.com/docs/height
if we need custom height later , we can add them to `tailwind.config.js` file, never do -> `h-[size]` it's a bad practice.

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -1,11 +1,6 @@
 export default function Footer() {
   return (
-    <div
-      className="flex justify-between items-center m-2 pl-3 pr-3 pt-2 pb-2"
-      style={{
-        background: "linear-gradient(to right, #ae2996, #825fc3, #519af4)",
-      }}
-    >
+    <div className="fixed inset-x-0 bottom-0 flex justify-between items-center m-2 pl-3 pr-3 pt-2 pb-2 bg-[linear-gradient(to_right,#ae2996,#825fc3,#519af4)]">
```
Comment (som-matrix) [inline src/components/Footer.tsx]: move this linear gradient to tailwind config file OR check if you can use default gradients provided by tailwind here -> https://tailwindcss.com/docs/background-image#linear-gradients

File: src/components/Header.tsx [hunk 3]
```diff
@@ -4,8 +4,7 @@ export default function Header() {
       <img src="/images/header-logo.png" alt="logo" className="h-6 w-6" />
       <div className="flex items-center gap-2">
         <button
-          className="p-2 rounded-full"
-          style={{ backgroundColor: "#1f1f1f" }}
+          className="p-2 rounded-full bg-[#1f1f1f]"
```
Comment (som-matrix) [inline src/components/Header.tsx]: again custom background colors , move them to tailwind config file OR use default colors -> https://tailwindcss.com/docs/background-color

File: src/components/main-panel/MainPanelFooter.tsx [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { footerData } from "./footerData";
+import { footerData } from "../../data/footerData";
```
Comment (som-matrix) [inline src/components/main-panel/MainPanelFooter.tsx]: relative imports looks messy when there are nested folders , use absolute imports instead.
check how to use absolute imports in react i think you need to configure your `tsconfig.json` file.

relative imports -> `"../../data/footerData"`
absolute imports -> `"data/footerData"`

File: src/components/main-panel/MainPanelCards.tsx [hunk 5]
```diff
@@ -16,16 +16,14 @@ export default function MainPanelCards() {
...
-                    hoveredCard?.itemIndex === itemIndex
-                      ? "rgba(209, 213, 219, 0.1)"
-                      : "",
-                }}
+                className={`p-2 rounded-md cursor-pointer relative ${
+                  hoveredCard?.sectionIndex === sectionIndex &&
+                  hoveredCard?.itemIndex === itemIndex
+                    ? "bg-gray-200/10"
+                    : ""
+                }`}
```
Comment (som-matrix) [inline src/components/main-panel/MainPanelCards.tsx]: use this package -> https://www.npmjs.com/package/clsx to handle dynamic classnames

**Review by som-matrix — CHANGES_REQUESTED** [review]
Please use tailwind default utility classes for height , width , background color , lineair graident in rest of the places

File: src/App.tsx (hunk 1 above)
Comment (jstsanam) [reply src/App.tsx]: Okay got it.

File: src/components/main-panel/MainPanelFooter.tsx (hunk 4 above)
Comment (jstsanam) [reply src/components/main-panel/MainPanelFooter.tsx]: Okay, will have a look into it.

File: tailwind.config.js [hunk 6]
```diff
@@ -2,7 +2,14 @@
 module.exports = {
   content: ["./src/**/*.{js,jsx,ts,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      height: {
+        '29': '29rem',
+      },
+      fontSize: {
+        'xxs': '0.5rem',
```
Comment (som-matrix) [inline tailwind.config.js:10]: no this doesn't looks goods keep it `0.5: 0.5em` is better , please make this change in a followup PR

File: src/components/Footer.tsx [hunk 7]
```diff
@@ -21,4 +16,4 @@ export default function Footer() {
       </button>
     </div>
   );
-}
+}
```
Comment (som-matrix) [inline src/components/Footer.tsx:19]: add empty line below , make this change in a follow up PR

**Review by som-matrix — APPROVED** [review]
Added minor comments , please make the changes in a PR or you can directly push to main no issues

**Pull request merged**